### PR TITLE
Feature/add defaults

### PR DESCRIPTION
**Pull request by Tom-Ritense**

### CODE REVIEW

**Review by PeterK-Ritense — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# All text files should have the "lf" (Unix) line endings
+* text eol=lf
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.java text
```
Comment (ivo-ritense) [inline .gitattributes:6]: should we add .kt?

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.6.3"
```
Comment (ivo-ritense) [inline build.gradle.kts:4]: official valtimo support is still 2.5.3

File: docker-compose-integration-test.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+version: '3.9'
+services:
+
+    allnex-test-db:
```
Comment (ivo-ritense) [inline docker-compose-integration-test.yml:4]: clean up name

File: docker-compose-integration-test.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        image: mysql:8
```
Comment (ivo-ritense) [inline docker-compose-integration-test.yml:5]: official support is 5.7.

also, i would like to switch over to postgres asap

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+version: "3.9"
+
+services:
+
+    # To fix 'Out of sort memory, consider increasing server sort buffer size' adjust value of 'sort-buffer-size'
```
Comment (ivo-ritense) [inline docker-compose.yml:5]: mainly an issue on mysql 8, for some reason 5.7 does this differently.

File: gradle.properties [hunk 6]
```diff
@@ -0,0 +1,10 @@
+projectName=give-me-a-name-valtimo-implementation
+#org.gradle.jvmargs=-Xms128m -Xmx1024m -XX:+CMSClassUnloadingEnabled
```
Comment (ivo-ritense) [inline gradle.properties]: Should we clean up all unused properties?

File: gradle/wrapper/gradle-wrapper.properties [hunk 7]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```
Comment (ivo-ritense) [inline gradle/wrapper/gradle-wrapper.properties:1]: I always wondered why we need to commit all this gradle junk if it's downloaded anyway.

File: build.gradle.kts (hunk 2 above)
Comment (Tom-Ritense) [reply build.gradle.kts]: kinnd of agree kind of want to upgrade

File: .gitattributes (hunk 1 above)
Comment (Tom-Ritense) [reply .gitattributes:6]: Dunno wat dit is eigenlijk

File: docker-compose-integration-test.yml (hunk 4 above)
Comment (Tom-Ritense) [reply docker-compose-integration-test.yml:5]: fine

File: docker-compose.yml (hunk 5 above)
Comment (Tom-Ritense) [reply docker-compose.yml]: ok

File: gradle/wrapper/gradle-wrapper.properties (hunk 7 above)
Comment (Tom-Ritense) [reply gradle/wrapper/gradle-wrapper.properties:1]: hmm i don't mind publishing the whore source

Comment (Tom-Ritense) [reply gradle/wrapper/gradle-wrapper.properties:1]: Having the source in the project speeds up the build time

**Pull request merged**